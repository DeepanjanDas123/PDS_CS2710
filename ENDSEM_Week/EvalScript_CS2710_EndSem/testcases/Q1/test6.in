BuildTreeFromFrequencyTable
22
z 38
W 35
p 44
H 45
s 15
9 43
2 35
w 32
D 35
L 19
4 11
P 11
E 24
C 13
O 1
U 29
k 40
b 40
j 36
n 31
Q 34
B 27
Quit
