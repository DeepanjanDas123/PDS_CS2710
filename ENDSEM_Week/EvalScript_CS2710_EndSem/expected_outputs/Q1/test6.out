((((n,w),(Q,2)),((D,W),(j,z))),(((b,k),((L,(P,(O,4))),9)),((p,H),((E,B),((C,s),U)))));
